### PR TITLE
Added Sugar ORM classes

### PR DESCRIPTION
**Pull request by divyanshkul**
I have added sugar ORM classes, do have a look at it. Also tried to add a junit test class but your guidance will be needed on that.

### CODE REVIEW

File: app/src/main/java/com/community/jboss/contactgroups/Contact.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import com.orm.SugarRecord;
+import java.util.UUID;
+import java.util.List;
+
+/**
+ * Created by Divyansh on 09-12-2017.
+ */
+
+public class Contact extends SugarRecord <Contact>{
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/contactgroups/Contact.java]: As class is Contact we don't need to add contact at every field name

Comment (divyanshkul) [reply app/src/main/java/com/community/jboss/contactgroups/Contact.java]: okay thanks @garg-anuj and the rest is okay?

**Comment by divyanshkul** [comment]
@garg-anuj please review the changes now.

**Comment by garg-anuj** [comment]
Thanks for your awesome work @divyanshkul 🎈 🌮 🎈 🎉

**Pull request closed without merging**